setup.py

```diff
@@ -51,14 +51,14 @@ def read_long_description():
     python_requires=">=3.7",
     install_requires=[
         "transformers>=4.11.3",
```

Comment:
Set here fixed `transformers` version and other libs: (`datasets`, datasets[vision]` and other libs that have '>=' instead of '='). Use versions that work in our local env.

Reply:
@GrzegorzR add here also:
```
datasets==2.12.0
```
I had problems with env setup without fixing versions of both (datasets and datasets[vision]).